m4opt/missions/_lsst/data/README.rst

```diff
@@ -0,0 +1,19 @@
+Reproduced Data from lsstCamSim.yaml
+====================================
+
+The contents of this directory are reproduced from the following source:
+`lsstCamSim.yaml <https://github.com/lsst/obs_lsst/blob/main/policy/lsstCamSim.yaml>`_.
+
+1. Extracting data from `lsstCamSim.yaml` is computationally expensive.
+   To optimize processing, we extract only relevant information, focusing on science detectors
+   while excluding wavefront and guide sensors. The remaining 189 detectors are stored in an
```

Comment:
No, we extract only the science sensors because we only care about targets that are within the science field of view.